run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# To make double clicking cbscript files work,
+# modify the defaults of your desktop environment.
+#
+# To generate blocks.json, via command prompt at the minecraft server:
+# java -DbundlerMainClass=net.minecraft.data.Main -jar {jar_path} --server --reports
+
+echo "CBScript 1.20"
+
+if [ $# -eq 0 ]; then
+    echo "No script provided."
+    exit
+fi
+
+if ! [ -f $1 ]; then
+    echo "Please provide a valid file."
+    exit
+fi
+
+echo -ne "\033]0;$(basename $1)\007"
+pushd $(dirname $0)
+python3 compile.py $1
```

Comment:
What if Python 3 isn't called `python3` on their system?

Reply:
Do you suggest I use just `python` then? Or check to see if 3 is installed?

Reply:
Nah, it's probably fine to use `python3`. If it's not installed it will error, and if it is installed it's 99% of the time going to have an alias of `python3`.

Reply:
Eh, it's probably fine. If they don't have `python3` installed that means they probably don't have well, python version 3 installed. It is very nonstandard and strange for them to have `python3` installed under the alias `python` without the subsequent `python3` command as-well.

Reply:
I agree; it's fine.